lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -558,9 +555,6 @@ def autoscale_view(self, tight=None, scalex=True, scaley=True,
             self.set_xbound(x0, x1)
 
         if scaley and self._autoscaleYon:
-            yshared = self._shared_y_axes.get_siblings(self)
```

Comment:
Are we sure that these have no side-effects?

Reply:
I've just gone back and traced through carefully - it *does* have the effect of `self._shared_y_axes.clean()`, so I've replaced the block with that.  Not sure if this is actually needed, but better safe than sorry!